### PR TITLE
fix #246

### LINKED ISSUE #246: James Harris showing $114,769 Not Stated but Expenses shows only $17,322
screenshot from 2017-01-17 20-22-10
screenshot from 2017-01-17 20-21-59

cc @sfdoran

### PR DESCRIPTION
**Pull request by mikeubell**
I have fixed up the candidate page to add IEC expenses.  
I added "see-more" links to the records of supporting and opposing expenses but was not able to get close to having those work, so I did not include my attempts at the supporting changes in appMain.

### CODE REVIEW

File: app/components/common/candidate/candidate_page.html [hunk 1]
```diff
@@ -64,4 +56,30 @@
       <odca-money-bar-chart value="{{ amount }}" max="{{ $ctrl.supporting.total_expenditures }}" format="money" color="red" ng-repeat-end></odca-money-bar-chart>
     </div>
   </div>
+
+  <div ng-if="$ctrl.supporting.supporting_money.supporting_expenditures" class="divided-section is-off-screen" when-visible="$ctrl.onVisible">
+    <h3>
+      Expenditures by Independant Committees supporting:
```
Comment (tdooner) [inline app/components/common/candidate/candidate_page.html]: spelling bug: "Independent"

File: app/components/common/candidate/candidate_page.html [hunk 2]
```diff
@@ -64,4 +56,30 @@
...
+    </div>
+    <div>
+      <div class="component__heading" ng-repeat-start="(label, amount) in $ctrl.supporting.supporting_money.supporting_by_type">{{ label }}</div>
+      <odca-money-bar-chart value="{{ amount }}" max="{{ $ctrl.supporting.supporting_money.supporting_expenditures }}" format="money" color="red" ng-repeat-end></odca-money-bar-chart>
+    </div>
+  </div>
+
+  <div ng-if="$ctrl.candidate.opposing_money.opposing_expenditures">
+   <h3>
+      Expenditures by Independant Committees opposing:
```
Comment (tdooner) [inline app/components/common/candidate/candidate_page.html]: (here too)

File: app/components/common/candidate/candidate_page.html [hunk 3]
```diff
@@ -64,4 +56,30 @@
...
+    </div>
+  </div>
+
+  <div ng-if="$ctrl.candidate.opposing_money.opposing_expenditures">
+   <h3>
+      Expenditures by Independent Committees opposing:
+      {{ $ctrl.candidate.opposing_money.opposing_expenditures | dollar }}
+   </h3>
+    <div class="see-more" >
+      <a class="see-more__content" ui-sref="appMain.committee.opposing({filer_id: $ctrl.candidate.filer_id})">See opposing <span class="text-no-break">committees<span class="fa fa-chevron-right"></span></span></a>
```
Comment (adborden) [inline app/components/common/candidate/candidate_page.html]: So to create this new state, you'll want to copy from [appMain.committee.main](https://github.com/caciviclab/disclosure-frontend/blob/974452ee51e401bc3672cc426277376c583746a9/app/components/appMainModule/committeePageModule/committeePageRoutes.js#L42) and then you'll want to add to it to automatically resolve the opposing data you need. If you look at the [parent state](https://github.com/caciviclab/disclosure-frontend/blob/master/app/components/appMainModule/committeePageModule/committeePageRoutes.js#L17), this is where the data is coming from for the existing contributions page. A quick refactor will make it a little more striaghtforward to copy/paste/edit.

Comment (adborden) [inline app/components/common/candidate/candidate_page.html]: I just pushed a [feature/opposing-expenditures](https://github.com/caciviclab/disclosure-frontend/compare/feature/opposing-expenditures) branch with the refactor. That might be a better place to start.

**Pull request closed without merging**